x/incentive/expected_keepers.go

```diff
@@ -20,4 +20,5 @@ type LeverageKeeper interface {
 	GetCollateralAmount(ctx sdk.Context, borrowerAddr sdk.AccAddress, denom string) sdk.Coin
 	IsAcceptedUToken(ctx sdk.Context, uTokenDenom string) bool
 	AssertNotBlacklisted(ctx sdk.Context, denom string) error
+	// TODO: hooks
```

Comment:
Similar to how oracle module registers leverage hooks to be run after adding tokens, incentive module will need to register leverage hooks to be run after liquidating or decollateralizing assets.

Reply:
Is this to remove user from a plan?

Reply:
Possibly - it can either instantly unbond bonded tokens, partially complete an ongoing unbonding by reducing its amount, or completely finish some ongoing unbondings.

In general, bonded tokens must be liquidated instantly regardless of their status in the incentive module so this will need to keep any numbers related to bonds or unbodings up to date.